parm/params_grib2_tbl_new

```diff
@@ -1217,7 +1207,6 @@
    0   2   192   1 VWSH
   10   4    17   0 WATDENA
   10   4    16   0 WATERDEN
-   2   0   236   1 WATERSA
```

Comment:
parames_grib2_tbl_new.text and params_grib2_table_new have the same definitions of the variables but in different old. Do you know why those variables (like WATERSA) are removed?